embedded-workshop-book/src/installation.md

```diff
@@ -43,6 +43,42 @@ $ rustup +stable component add llvm-tools-preview
 $ cargo install cargo-binutils
 ```
 
+## `nrf-recover`
+
+Some nRF52840 devices, specially older revisions, may have parts of their Flash memory locked. To unlock the memory use the `nrf-recover` tool.
+
+This is only relevant to the nRF52840 Development Kit. First connect the nRF52840 DK to your PC using micro-USB J2 (as done before) then run the following commands:
+
+``` console
+$ cargo install nrf-recover
+
+$ nrf-recover -y
```

Comment:
the issue with moving this one here is that you need to do the Zadig thing (on Windows) before you run `nrf-recover` and now that comes after this section.

Reply:
aaah grr. I've reordered again and grouped them by OS specific deps, rust, python and nrf tools